txpool/txpool.go

```diff
@@ -72,13 +88,13 @@ func NewTxPool(
 	network *network.Server,
 ) (*TxPool, error) {
 	txPool := &TxPool{
-		logger:     logger.Named("txpool"),
-		store:      store,
-		idlePeriod: defaultIdlePeriod,
-		queue:      make(map[types.Address]*txQueue),
-		sorted:     newTxPriceHeap(),
-		sealing:    sealing,
-		forks:      forks,
+		logger:           logger.Named("txpool"),
+		store:            store,
+		idlePeriod:       defaultIdlePeriod,
+		accountTxHeapMap: make(map[types.Address]*txHeapWrapper),
+		validTxHeap:      newTxPriceHeap(),
```

Comment:
Heap is a data structure commonly used to implement queues (abstract data structure).
It's perhaps easier to reason about what our intention is if we talk about queues rather than heaps.

Also, I don't think that it is helpful if we include the explicit type in the name of the variable (e.g. map, etc.)

My suggestion is to rename `accountTxHeapMap` to `accountQueues` and `validTxHeap` to `queue`/`pendingQueue`.

What do you think?

Reply:
I completely agree. I wasn't exactly sure what to name them as well, but I guess it can be figured out from the comments as well as the renamed fields 👍 
44c44bb